core/blockinfo/service.go

```diff
@@ -0,0 +1,50 @@
+package blockinfo
```

Comment:
Is this blockinfo guaranteed to be consistent across consensus engines and between comet and roll kit? Is this guaranteed to be stable across all versions of comet? If not, we should put it in a different package, maybe even outside of core